kong/include/kong/pluginsocket.proto

```diff
@@ -101,7 +101,7 @@ message String {
 
 message ExitArgs {
     int32 status = 1;
-    string body = 2;
+    bytes body = 2;
     google.protobuf.Struct headers = 3;
```

Comment:
for backward compatibilty, we might need to add a new field, and rename the old field but keep same id 🤔 ?

Reply:
🤦 We are using `String`, which is defined below, as many RPC's parameters and return types:
```protobuf
message String {
    string v = 1;
}
```
It's not easy to keep backward compatibility for this.

I come up with a hack that we could add a type like:
```protobuf
message ByteString {
    string v_lagacy = 1;
    bytes v = 2;
}
```
and we use that as the type for `get/setRawBody`. The older version of GO PDK would treat the type as String, and they are compatible.

Reply:
Finally we decided not to have these compatibility.